server/src/main/java/org/apache/cloudstack/user/PasswordReset.java

```diff
@@ -0,0 +1,55 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.cloudstack.user;
+
+import com.cloud.user.UserAccount;
+import org.apache.cloudstack.framework.config.ConfigKey;
+
+public interface PasswordReset {
+    ConfigKey<Long> PasswordResetTtl = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED, Long.class,
+            "password.reset.ttl", "30",
+            "Password reset ttl in minutes", true, ConfigKey.Scope.Global);
+
+    ConfigKey<String> PasswordResetEmailSender = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED,
+            String.class, "password.reset.email.sender", null,
+            "Password reset email sender", true, ConfigKey.Scope.Global);
+
+    ConfigKey<String> PasswordResetSMTPHost = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED,
```

Comment:
Does it makes sense to keep all the settings to the Account Scope ?

Reply:
IMO, it should be Global. These settings should be managed by the root admin only.

Reply:
Even if we keep it Account/Domain level also they will be controlled by root admin. Different domains may have different set of configurations, default will be global anyways.

Reply:
If we keep it a account/domain level, a domain admin will be able to update these settings. I am not able to think of a scenario where a domain or an account might have different smtp host configuration for sending the password reset email.

Reply:
I think there can be cases like that, if the CloudStack deployment itself is used for public clouds, each domain is definitely needs to have different configurations.